vignettes/TET2.Rmd

```diff
@@ -111,6 +114,8 @@ fit1 <- eBayes(lmFit(exprs(DNAme), design1))
                                            coef=grep("IDH", colnames(design1)), 
                                            p.value=0.05, # change if you like 
                                            number=Inf)))
+#if you're having trouble running this entire chunk or receiving error "'pData': object 'DNAme' not found", run design1 chunk of code first and the rest of the chunk should follow suit. I don't know why that is or if it is just me.
+
```

Comment:
Yeah I think the fit1 is running a Bayesian analysis based on "design1" so you need to have design 1 set first. We repeated this three times, each time setting up a design for the fit to be based on. I think. Maybe. Probably.